library/globals.inc.php

```diff
@@ -949,14 +949,14 @@
       xl('Exclude Statements from Printing'),
       'bool',                           // data type
       '0',                              // default = false
-      xl('This will use the Following settings to Exclude Patient Statements from Printing.')
+      xl('This will enable the Ability to Exclude Patient Statements from Printing.')
```

Comment:
Would the above two statements then cover the exclusion options you plan to add in the future?